test/README.md

```diff
@@ -379,3 +379,12 @@ The realpath of the 'tmp' directory.
 * return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
 
 Name of the temp directory used by tests.
+
+### WPT
```

Comment:
Er, although in other instances, parentheses are used to show arguments. So maybe I'll just mention it in the text, but try to do it earlier.